package-lock.json

```diff
@@ -213,7 +213,7 @@
         },
         "chalk": {
           "version": "1.1.3",
-          "resolved": "https://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
+          "resolved": "http://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
```

Comment:
My computer has been doing this for _ages_ now and I can't figure out why. Very annoying.